Remotion/Validation/UnitTests/Implementation/FallbackValidationMessageFactoryTest.cs

```diff
@@ -15,23 +15,84 @@
 // along with re-motion; if not, see http://www.gnu.org/licenses.
 // 
 using System;
+using Moq;
 using NUnit.Framework;
+using Remotion.Reflection;
+using Remotion.Validation.Implementation;
+using Remotion.Validation.UnitTests.TestDomain;
+using Remotion.Validation.Validators;
 
 namespace Remotion.Validation.UnitTests.Implementation
 {
   [TestFixture]
   public class FallbackValidationMessageFactoryTest
   {
     [Test]
-    [Ignore("RM-5906")]
     public void CreateValidationMessageForPropertyValidator ()
     {
+      var propertyValidatorMock = new Mock<IPropertyValidator>();
+      var validatedPropertyMock = new Mock<IPropertyInformation>();
+
+      var fallbackFactory = new FallbackValidationMessageFactory();
+      var result = fallbackFactory.CreateValidationMessageForPropertyValidator(propertyValidatorMock.Object, validatedPropertyMock.Object);
+
+      Assert.That(result, Is.Not.Null);
+      Assert.That(result, Is.InstanceOf<InvariantValidationMessage>());
```

Comment:
```suggestion
      Assert.That(result, Is.InstanceOf<InvariantValidationMessage>());
```
In this scenario, the null-check is redundant. The type check would complain with a meaningful message anyway.